storage/oasis/nodeapi/api.go

```diff
@@ -6,12 +6,13 @@ import (
 	"time"
 
 	coreCommon "github.com/oasisprotocol/oasis-core/go/common"
+
 	hash "github.com/oasisprotocol/oasis-core/go/common/crypto/hash"
 	"github.com/oasisprotocol/oasis-core/go/common/crypto/signature"
 	"github.com/oasisprotocol/oasis-core/go/common/errors"
-	node "github.com/oasisprotocol/oasis-core/go/common/node"
 
 	beacon "github.com/oasisprotocol/nexus/coreapi/v22.2.11/beacon/api"
+	node "github.com/oasisprotocol/nexus/coreapi/v22.2.11/common/node"
```

Comment:
was this from #548?

Reply:
Yep. I put it in the wrong commit apparently; apologies. #548 is merged now, so leaving this one as-is.